README.md

```diff
@@ -1,52 +1,21 @@
 
+## source{d} Ghost Theme Assets, Dependencies & Configurations
 
-## source{d} Ghost Theme
+This repo gathers source{d} Ghost Theme configuration instructions, depedent assets and contributing guidelines, and the Theme itself. 
 
-This theme is a customization for source{d} based on a blank template for Ghost.
+### Instructions
 
-### All Ghost features
+- To update the source{d} Ghost Theme, the entire [ghost-theme directory](https://github.com/src-d/sourced.ghost.io/tree/master/ghost-theme) should be ziped and uploaded/updated on Ghost -> Settings -> Design.
 
-It includes all the [Ghost](https://ghost.org/) functionalities a theme can provide, in a modular implementation so it is easy to customize those functionalities or get rid of them.
+- The [docs directory](https://github.com/src-d/sourced.ghost.io/tree/master/docs/README.md) has the assets that are depentents on source{d} Landing, in order to brand-wise comply with the current/updated Landing Identity, [Landing's External Dependents](https://github.com/src-d/landing/blob/master/CONTRIBUTING.md#external-dependents) must be taken into cosideration.
 
-Probably you will not want everything [Ghost](https://ghost.org/) shows on a theme, but trust me when I say it is easy to remove those stuff you do not want to see instead of add them if they are missing in the first place.
+- You can find Ghost Theme Settings Instructions on [ghost-settigns directory](https://github.com/src-d/sourced.ghost.io/tree/master/ghost-settings/README.md).
 
-### Code separated into different .hbs files
+- Current source{d} Ghost Theme Styles are defined [on this directory](https://github.com/src-d/sourced.ghost.io/tree/master/assets/scss) and use some of Landign's Styles, mainly of the Header & Footer. You can find bellow some forward thinking on this matter.
```

Comment:
it does not exist `assets/scss` under the branch you're pull requesting.
It seems you moved it into `ghost-theme`, so it would be now `ghost-theme/assets/scss/`

Reply:
Thanks @dpordomingo Fixed!